src/treeChooser/treeChooser.html

```diff
@@ -1,5 +1,5 @@
 <div class="treeChooser">
-  <div class="treeChooser-input"
+  <div class="treeChooser-input" tabindex="0"
```

Comment:
Is this going to cause an extra tab stop? So if I'm in the field to the right, do I tab once to land on this div, and then again to land in the actual input?

Reply:
Didn't find extra tab stop. Works correctly